lib/smart_proxy_dhcp_infoblox/fixed_address_crud.rb

```diff
@@ -5,16 +5,34 @@ module ::Proxy::DHCP::Infoblox
   class FixedAddressCRUD < CommonCRUD
     attr_reader :network_view
 
-    def initialize(connection, network_view)
+    def initialize(connection, network_view, used_ips_search_type)
       @memoized_hosts = []
       @memoized_condition = nil
       @network_view = network_view
-      super(connection)
+      @used_ips_search_type = used_ips_search_type
```

Comment:
Rather than declaring this again, I think you can rely on the parent class (you do call `super()` correctly).